app/javascript/lib/scp-api.jsx

```diff
@@ -361,6 +361,37 @@ export async function fetchBucketFile(bucketName, filePath, maxBytes=null, mock=
 }
 
 
+
+
+/**
+ * Download a file retreived from a Google Bucket
+ *
+ * @param {String} bucketId bucket id
+ * @param {String} filePath path to file in bucket
+*/
+export async function downloadBucketFile(bucketId, filePath) {
+  // Fetch the data from the bucket
+  const data = await fetchBucketFile(bucketId, filePath)
+
+  // Convert data to a blob (standard for non-same-origin downloads)
+  const dataBlob = await data.blob()
+
+  // Create an element with an anchor link and connect this to the blob
+  const element = document.createElement('a')
+  element.href = URL.createObjectURL(dataBlob)
+
+  // name the file and indicate it should download
+  const filePathAsArr = filePath.split('/')
+  element.download = `${filePathAsArr.pop()}`
```

Comment:
`pop` ought to return a string, and we're not interpolating anything else, so it seems we can simplify to:

```suggestion
  element.download = filePathAsArr.pop()
```

For reference:
```
foo = 'abc/efg/123'
'abc/efg/123'
bar = foo.split('/')
(3) ['abc', 'efg', '123']
bar.pop()
'123'
```

Reply:
Yeah that seems safe, the reference code is nice thanks